client/state/selectors.js

```diff
@@ -38,7 +38,22 @@ const getRawFormErrors = ( schema, data ) => {
 
 export const getFormErrors = createSelector(
 	( state ) => state.form.errors,
-	( state, props ) => props.schema,
+	( state, schema ) => schema,
 	( state ) => state.settings,
 	( errors, schema, data ) => removeErrorDataPathRoot( errors || getRawFormErrors( schema, data ) )
 );
+
+export const getStepFormErrors = createSelector(
+	( state ) => state,
+	( state, schema ) => schema,
```

Comment:
What do you mean? This is how `reselect` works. Really, it passes the 3 parameters (`state`, `schema` and `layout`) to all the functions, but since they are not needed I just omitted them.